### PR TITLE
[BML-10]Update BillML repo to parse all bill data and create, upload full datasets

### PR DESCRIPTION
**Pull request by BorodaUA**
Hi 👋 team, please review changes that I made in scope of following task(s):

- [BML-10]Update BillML repo to parse all bill data and create, upload full datasets

Here what is done:

- added cutom logger 
- refactored project structure so it can be included as sub repo in https://github.com/dreamproit/BillML project
- speed up bill status and bill text downloads with multiprocessing
- added CLI script to show info about local bills data and missing bill status, bill text data with ability to download missing data(very handy after project initial downloads)

TODO:
- use black, isort, flake8 for whole project after this PR review.

### CODE REVIEW

File: core/tasks/bill_info.py [hunk 1]
```diff
@@ -397,7 +408,14 @@ def keep_action(item, closure):
 
         keep = True
         if closure['prev']:
-            if item['sourceSystem'].get('code') == "9":
+            # try:
```
Comment (alexbojko) [inline core/tasks/bill_info.py]: rm commented code

File: core/tasks/bill_info.py [hunk 2]
```diff
@@ -863,7 +882,9 @@ def parse_bill_action(action_dict, prev_status, bill_id, title):
             bill_item = bill_item.lower().replace(".", "").replace(" ", "").split(",")
             if bill_item[0] == (bill_type + number):
                 as_amended = len(bill_item) > 1
-        if as_amended is None: raise ValueError("Did not find bill in list: " + line)
+        if as_amended is None: 
+            # raise ValueError("Did not find bill in list: " + line)
```
Comment (alexbojko) [inline core/tasks/bill_info.py]: do we need it?

File: core/tasks/bill_info.py (hunk 1 above)
Comment (BorodaUA) [reply core/tasks/bill_info.py]: Good catch, removed in the new commit

File: core/tasks/bill_info.py (hunk 2 above)
Comment (BorodaUA) [reply core/tasks/bill_info.py]: Good catch, removed in the new commit

**Review by alexbojko — APPROVED** [review]

**Pull request merged**